src/main/java/org/prebid/server/analytics/AnalyticsReporterDelegator.java

```diff
@@ -168,4 +181,43 @@ private static ObjectNode prepareAnalytics(ObjectNode analytics, String adapterN
 
         return !analyticsNodeCopy.isEmpty() ? analyticsNodeCopy : null;
     }
+
+    private <T> void processEventWithResult(T event, AnalyticsReporter analyticsReporter) {
```

Comment:
The name is too generic. Pls consider to use something like `processEventByReporter(AnalyticsReporter analyticsReporter, T event)`.
Note the reporter becomes first argument since we've mentioned in method name.